core/rpc/src/author/mod.rs

```diff
@@ -61,12 +67,14 @@ pub struct Author<B, E, P, RA> where P: PoolChainApi + Sync + Send + 'static {
 impl<B, E, P, RA> Author<B, E, P, RA> where P: PoolChainApi + Sync + Send + 'static {
 	/// Create new instance of Authoring API.
 	pub fn new(
+		executor: Arc<dyn Executor<Box<dyn Future<Item = (), Error = ()> + Send>> + Send + Sync>,
```

Comment:
Should we typedef this? It's used in `api/src/subscriptions.rs` as well and looks pretty nasty.

Reply:
I've added a typedef, but then got rid of `executor` field in `Author` as you've suggested. Typedef is still there - if you thing it isn't required now, I can revert